src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -62,7 +62,9 @@ public function handle($request, Closure $next)
         // Again, if the session has been configured we will need to close out the session
         // so that the attributes may be persisted to some storage medium. We will also
         // add the session identifier cookie to the application response headers now.
-        $this->saveSession($request);
+        if ($this->manager->driver()->isDirty()) {
```

Comment:
@imanghafoori1 The `isDirty`  method should be added to the `Session` interface (which is a breaking change and should target master), otherwise this will break for all other session store implementations. If this fix must go into 5.8 too then we should add a `method_exists` check here before calling the `isDirty` method.

Reply:
thank for mentioning that.
I missed it